src/ensembl/src/shared/components/instant-download/instant-download-transcript/InstantDownloadTranscriptVisualisation.tsx

```diff
@@ -47,10 +47,18 @@ export type Props = {
 const InstantDownloadTranscriptVisualisation = (props: Props) => {
   const exonsCount = 5;
   const exonHeight = 5;
-  const proteinHeight = 3;
+  const exonWidth = Math.floor(props.width / 7);
+  const proteinHeight = 4;
+  const proteinBlockWidth = proteinHeight;
+  const proteinBlockSpacing = 1;
+  const innerProteinExonBlocksCount = Math.round(
```

Comment:
innerProteinBlocksCount and outerProteinBlocksCount?

Reply:
Sorry, missed it! Done now.